PYME/cluster/HTTPDataServer.py

```diff
@@ -567,6 +567,8 @@ def send_head(self):
                 return self.list_h5(path)
             else:
                 return self.get_h5_part(path)
+        elif '.h5r' in self.path or '.hdf' in self.path:
```

Comment:
We need to test for extension **and** trailing slash - i.e. `'.h5r/` as we want a simple get on `somefile.h5r` to return the whole file.

Reply:
```suggestion
        elif '.h5r/' in self.path or '.hdf/' in self.path:
```